AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -58,6 +58,8 @@ disk.yandex.*##.mail360-banner
 @@||yastatic.net/daas/atom.js$domain=mail.yandex.by|mail.yandex.kz|mail.yandex.ru|mail.yandex.ua|mail.yandex.com|mail.yandex.com.tr
 !-----------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/pull/111386
+m.coupang.com###buy-button > #sdp-bottom-promotion-banner-sign-up
```

Comment:
```
m.coupang.com###buy-button > div#sdp-bottom-promotion-banner-sign-up
```
Also it's a popup, so you should move it to this file.

Reply:
Pushed.